src/posts/tools-for-research.md

```diff
@@ -0,0 +1,84 @@
+
+Chris Sacca—one of the most successful venture capitalists ever—insists that <a href="https://twitter.com/sacca/status/1134128910004772869?s=20" target="_blank" rel="noopener noreferrer">note taking is a foundational skill of every successful person he knows</a>.
+
+I can attest to the fact that since forming my own note taking and journaling habits, I'm far more confident in tackling complex subjects, and learn at a much faster rate. Note taking is one of the most important forms of <a href="https://rdjpalmer.com/week-39-2020-observation-is-my-superpower" target="_blank" rel="noopener noreferrer">observation</a>, which is a transformative skill in its own right.
+
+Roam itself helps you build on this through bi-directional linking, and an interface which gets out of your way and focuses you on exploring your thoughts. Don’t let my simple description fool you though, there’s way more to Roam than one novel feature.
+
+It’s not to everyone’s tastes. It’s a little ugly and rough round the edges. Thankfully, Nesslabs is to the rescue again, with <a href="https://nesslabs.com/how-to-choose-the-right-note-taking-app" target="_blank" rel="noopener noreferrer">How to choose the right note taking app</a>.
+
+## Index cards as bookmarks
+
+<a href="https://rdjpalmer.com/week-33-2020" target="_blank" rel="noopener noreferrer">Index cards and post-it notes</a> have turned into one of my go to weapons on a daily basis. From taking notes to deciding immediate priorities, just getting something down on paper is far more useful than trying to keep it in your head.
+
+Next time you’re reading a physical book, replace your bookmark with an index card. As you make your way through the book, highlight passages in the book and mark down the page numbers on the card. Add commentary as you go.
+
+When you're done with that session, write them up in Roam to reference later.
+
+If you’ve neat enough handwriting, you can skip the write up by using Google Lens. Take a picture of it, let Google Lens pick up on all the words, and copy it straight to your computer. Magic ✨
+
+## Hypothes.is ￫ ReadWise ￫ Roam
+
+The digital equivalent of the index card, <a href="https://web.hypothes.is/" target="_blank" rel="noopener noreferrer">Hypothes.is</a> let's you annotate and highlight almost any page on the web.
+
+By default, it saves all your highlights and annotations to a public feed where other users can comment and discuss them with you.
+
+For me, on it's own isn't all that beneficial, but combine it with <a href="https://readwise.io/" target="_blank" rel="noopener noreferrer">ReadWise</a> and you're on to a winner.
+
+ReadWise resurfaces your notes to you in a handy summary email every so often. Using a technique known as spaced repetition which is proven to help with recall.
+
+And the new Roam integration really levels the whole system up. Instead of keeping the notes to itself, ReadWise now puts your notes where they should be: in your note taking app.
```

Comment:
```suggestion
And the new Roam integration levels the whole system up. Instead of keeping the notes to itself, ReadWise now puts your notes where they should be: in your note taking app.
```